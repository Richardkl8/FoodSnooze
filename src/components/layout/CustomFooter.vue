<template>
    <footer>
        <div class="container">
            <custom-h3>
                FoodSnooze
            </custom-h3>
        </div>
    </footer>
</template>

<script>
    export default {
        name: 'Header',
    };
</script>

<style lang="scss" scoped>
    footer {
        margin-top: 3rem;
        background-color: #2DB396;
        width: 100%;
        height: 10rem;
    }
</style>
