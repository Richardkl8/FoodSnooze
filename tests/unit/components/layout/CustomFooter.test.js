import { shallowMount } from '@vue/test-utils';
import CustomFooter from '../../../../src/components/layout/CustomFooter.vue';
import stubs from '../../stubs/stubs';

describe('CustomFooter', () => {
    test('Should match snapshot', () => {
        const wrapper = shallowMount(CustomFooter, {
            stubs,
        });
        expect(wrapper).toMatchSnapshot();
    });
});
