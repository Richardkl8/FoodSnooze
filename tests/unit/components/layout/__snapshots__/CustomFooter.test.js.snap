// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomFooter Should match snapshot 1`] = `
<footer class="center-items">
  <div class="container">
    <custom-h3-stub class="text-white"><b>FoodSnooze</b><br>
      Who doesn't like to snooze?!
    </custom-h3-stub>
  </div>
</footer>
`;
